Synergism.css

```diff
@@ -2143,6 +2143,30 @@ nav li:hover {
     background-color: #333;
 }
 
+.autoOpensBtn {
+    position: absolute;
```

Comment:
doesn't need to be absolute positioned

Reply:
It wont move with zoom, dont worry

Reply:
Its no longer is absolute positioned, but no idea if my way is any better than absolute,
At least it still doesnt move text